index.html

```diff
@@ -197,7 +204,11 @@ <h2>Examples</h2>
 
             const codeRes = await fetch(`https://cdn.jsdelivr.net/gh/jcubic/chat-gpt@${hash}/${filename}`);
             if (!codeRes.ok) throw new Error('Failed to fetch bookmarklet code');
-            const code = await codeRes.text();
+            let code = await codeRes.text();
+
+            // Inject the saved CORS proxy URL by replacing the placeholder token
+            const proxy = localStorage.getItem('cors_proxy') || '';
+            code = code.replace(/__CORS_PROXY__/g, JSON.stringify(proxy));
```

Comment:
The logic is wrong. The code inject a string:

"https://

into "__CORS_PROXY__"

So you end up with `""<URL>""`.